pkg/agent/agent.go

```diff
@@ -29,21 +34,231 @@ import (
 	"github.com/nimbess/nimbess-agent/pkg/proto/cni"
 )
 
+// CNI Reply Values
+const (
+	CniOk                = 0
+	CniIncompatible      = 1
+	InvalidNetworkConfig = 2
+	ContainerNotExist    = 3
+	CniTryLater          = 11
+	DriverFailure        = 101
+)
+
+var ipAddr = net.IP{40, 0, 0, 0}
```

Comment:
40,0,0,0 - is that significant?

Reply:
no just something to use as hardcoded ipam until i add support for using CNI ipam

Reply:
i'll add a TODO remove this comment above it